lib/Distribution/Sle/15_current.pm

```diff
@@ -98,4 +100,8 @@ sub get_dns_server_setup {
     return YaST::DNSServer::Sle::DNSServerSetupController->new();
 }
 
+sub get_ntp_config {
```

Comment:
s/config/configuration/ (the preference is always whatever developers thought was best for wording, even if it is long), but yeah neat picking here :)